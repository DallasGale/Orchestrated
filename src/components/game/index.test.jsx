import React from 'react'
import { render } from 'enzyme'
import toJson from 'enzyme-to-json'
import Game from '.'

const props = {
  addOnClick: () => { },
  removeOnClick: () => { },
  price: 0,
  save: 0,
  title: 'Title',
  was: 0,
}

describe('<Game />', () => {
  test('it renders', () => {
    const Component = render(<Game {...props} />)
    expect(toJson(Component)).toMatchSnapshot()
  })
})
