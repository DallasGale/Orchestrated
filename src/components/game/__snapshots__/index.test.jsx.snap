// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Game /> it renders 1`] = `
<article
  class="sc-iwsKbI bCWdSU"
>
  <div
    class="sc-gZMcBi cdJWWr"
    style="background-image:url('https://images.unsplash.com/photo-1562819789-fd366076971a?ixlib=rb-1.2.1&auto=format&fit=crop&w=1650&q=80')"
  />
  <div
    class="sc-bZQynM jTXKpG"
  >
    <div
      class="sc-jzJRlG fcSxCN"
    >
      <h2
        class="sc-bwzfXH czBpvq"
      >
        Title
      </h2>
      <p
        class="sc-VigVT bNtpsr"
      >
        No description found.
      </p>
    </div>
  </div>
  <div
    class="sc-htoDjs gSrElP"
  />
  <div
    class="sc-bZQynM jTXKpG"
  >
    <div
      class="sc-fjdhpX kHyemL"
    >
      <div
        class="sc-jTzLTM fXFYMA"
      >
        $0
      </div>
      <small
        class="typography__align--right sc-cSHVUG hdhyAz"
      >
        Was $0, save $0
      </small>
    </div>
    <div
      class="sc-fjdhpX kHyemL"
    >
      <div>
        <button
          class="sc-ifAKCX ewMubd"
          type="button"
        >
          Add to cart
        </button>
      </div>
      <div
        class="typography__align--right"
      >
        <button
          class="sc-dnqmqq ltTAv"
          type="button"
        >
          Save to wishlist
        </button>
      </div>
    </div>
  </div>
</article>
`;
